homebrew_challenge/src/components/HeatMap/Heatmap.jsx

```diff
@@ -248,6 +273,8 @@ function Heatmap({
     );
   }
 
+  const dataset = getDataSet();
```

Comment:
Is this still needed?

Reply:
I have moved it into the body of the function where it is used